src/test/CMakeLists.txt

```diff
@@ -599,7 +599,7 @@ add_ceph_test(run-cli-tests ${CMAKE_CURRENT_SOURCE_DIR}/run-cli-tests)
 add_ceph_test(smoke.sh ${CMAKE_CURRENT_SOURCE_DIR}/smoke.sh)
 
 find_program(PROMTOOL_EXECUTABLE promtool)
-if(PROMTOOL_EXECUTABLE)
+if(EXISTS PROMTOOL_EXECUTABLE)
```

Comment:
quote from https://cmake.org/cmake/help/latest/command/if.html#basic-expressions

> True if the constant is 1, ON, YES, TRUE, Y, or a non-zero number. False if the constant is 0, OFF, NO, FALSE, N, IGNORE, NOTFOUND, the empty string, or ends in the suffix -NOTFOUND.

Reply:
Yup, that is what I read as well, but still my runs include the promtool tests.
And reevaluating the results it seems that we run into the docker-case.
So I'll remove the `EXIST` part.